python/packages/nisar/products/writers/GcovWriter.py

```diff
@@ -494,18 +494,26 @@ def populate_data_parameters(self):
         """
         Populate the data group `grids` of the GCOV product
         """
-        for frequency in self.freq_pols_dict.keys():
+        for frequency, pol_list in self.freq_pols_dict.items():
```

Comment:
```suggestion
        for frequency, _ in self.freq_pols_dict.items():
```
Looks like the variable `pol_list` is not used in this for loop.

Reply:
Thank you, @seongsujeong ! This was addressed in the commit https://github.com/isce-framework/isce3/pull/43/commits/245567e08abe9ae9d5d9b082686ca544f13a9f2e.